### ISSUE TITLE
Ways To Give - Wire Transfer

### ISSUE THREAD

**Issue by etrepum** [MissionBit/missionbit.org#16]
[Fundraise Page Language](https://docs.google.com/document/d/1u8DQEzqyiEisfDJYvH0WM1Tc1vCjogP7pv9Pdx3Qbng/edit)

> Please view our Wire Transfer Form for more information

Blocked on: Wire Transfer Form

**Comment by etrepum** [comment]
This is superseded by the new design
